README.md

```diff
@@ -187,7 +192,9 @@ in your terminal._
 You can specify the browser and Node.js versions by queries (case insensitive):
 
 * `defaults`: Browserslist’s default browsers
-  (`> 0.5%, last 2 versions, Firefox ESR, not dead`).
+  (`> 0.5%, last 2 versions, safe, not dead`).
+* `balanced`: target modern (updated) browsers (`last 2 versions, safe, not
+  unsafe, not < 0.1%`).
```

Comment:
I against removing non-popular browsers (`not < 0.1%`), because it will block new browsers coming to the market (nobody supports it because nobody use them / nobody uses them because nobody supports them).

Reply:
Please note that it only blocks browsers when they _ don't respect the standards_. From a webapp developer point of view, overloading each & every visitors with dozens if not hundreds Kos of polyfills only to support a browser that has under 0.1% market share & a low standard compliance is a big no-no - especially as bandwidth is a also an accessibility matter.

Reply:
> Please note that it only blocks browsers when they _ don't respect the standards_.

Why? It is the last query, it takes browsers from the right (`last 2 versions, safe, not unsafe`) and removes all `< 0.1%` browsers from there.

It will remove all non-popular browsers even if they support standards.

Reply:
(more generally, as explained in the link, the point of that proposal is to achieve a saner polyfills set by cutting off browsers that are expensive to support but that may not be part of sophisticated webapps targetted audience)

Reply:
> may not be part of sophisticated webapps targetted audience

Browserslist is about keeping diversity. I am from Russia. We all used the Opera 10 years ago. But nobody used Opera outside of Russia. And most of the websites did not support us.

I created Browserslist to avoid removing minor browsers just because Western developers do not know about them.

I understand that you want to re-think the default browser query and created a good alternative. Splitting the audience to “sophisticated” and non-sophisticated will not work as argument for me. We must keep browsers popular in Afrika and China.

Reply:
> It will remove all non-popular browsers even if they support standards.

The fact that a browser is not in the list doesn't means that it won't be supported. For example, if your list includes Chrome 80 then chances are Chrome 70+ will all work because they have the same level of support for the standards. Likewise, any browser following the standards will work.

Practically, the fact that a browser is not included in the list means that the webapp won't include polyfills that are required only by that browser.

Reply:
Browserslist is also used in `Your browser is not supported` banners.

Let’s speak honestly, what browsers exactly do you want to remove by this query?

Reply:
I just said it - anything that has a very low audience but require dozens of Kos of polyfills. That's a dynamic rule, that's not aimed at a brand in particular.

Reply:
1. `not < 0.1%` != “a very low audience but require dozens of Kos of polyfills”, because you remove browsers with good spec support
2. I do not think that it is a good ethical choice. I do not want to block minorities from the Internet. In the `not dead` we at least promoting Internet security.

Suggest different criteria.